site/ru/README.md

```diff
@@ -1,25 +1,30 @@
-# Не Официальный Перевод
+# Неофициальный перевод
 
-Вся информация в этом разделе переведена при помощи русскоговорящего Tensorflow
-сообщества на общественных началах. По скольку этот перевод не является
+Вся информация в этом разделе переведена с помощью русскоговорящего Tensorflow
```

Comment:
What do these changes do?

Reply:
Some minor improves to achieve better wording in the Russian block of text, orthography fixes mostly